packages/widget-playground-vite/package.json

```diff
@@ -44,7 +44,7 @@
     "source-map-explorer": "^2.5.3",
     "typescript": "^5.4.5",
     "vite": "^5.2.11",
-    "vite-plugin-node-polyfills": "^0.21.0",
+    "vite-plugin-node-polyfills": "^0.22.0",
```

Comment:
Had to bump this as it seemed to be causing problems with the Vite chunk optimisation